inventory_reporting/reports/inventory_report.py

```diff
@@ -337,3 +346,52 @@ def generate_consumable_report(self, workbook, wizard):
                     16,
                     valuation.stock_move_id.analytic_account_names or "",
                 )
+
+    def generate_summary_report(self, workbook, wizard):
+        ws = workbook.add_worksheet(_("Inventory Summary"))
+        valuation_obj = self.env["stock.valuation.layer"]
+        product_categories = self.get_product_categories()
+        base_storable_domain, storable_categories = self.get_storable_categories(wizard)
+        headers = [
+            _("Product Category"),
+            _("Inventory Total Value"),
+            _("Report Category"),
+            _("Report Total Value"),
```

Comment:
```suggestion
            _("Stock Move Category"),
            _("Stock Move Category Total Value"),
```
I feel Report is vague.
How do you think?

Reply:
How about Storable Category or Valuation Category?

Reply:
I think Inventory Operation Type is better.